app/components/dashboard/hypertension/overdue_patients_called_table_component.html.erb

```diff
@@ -64,13 +64,13 @@
                   <%= count_by_period[period] %>
                 </td>
                 <% if facility? %>
-                  <td class="type-percent" data-sort-column-key="population-coverage" data-toggle="tooltip" title="" data-original-title="<%=patients_called_tooltip(count_by_period[period], overdue_patients(region, period)) %>">
+                  <td class="type-percent" data-sort-column-key="population-coverage" data-sort="<%=percentage(count_by_period[period], overdue_patients(region, period))%>" data-toggle="tooltip" title="" data-original-title="<%=patients_called_tooltip(count_by_period[period], overdue_patients(region, period)) %>">
                     <em data-rate="<%=percentage(count_by_period[period], overdue_patients(region, period))%>">
                       <%=percentage_string(count_by_period[period], overdue_patients(region, period))%>
                     </em>
                   </td>
                 <% else %>
-                  <td class="type-percent" data-sort-column-key="population-coverage" data-toggle="tooltip" title="" data-original-title="<%=patients_called_tooltip(count_by_period[period], overdue_patients(row, period)) %>">
+                  <td class="type-percent" data-sort-column-key="population-coverage" data-sort="<%=percentage(count_by_period[period], overdue_patients(row, period))%>" data-toggle="tooltip" title="" data-original-title="<%=patients_called_tooltip(count_by_period[period], overdue_patients(row, period)) %>">
```

Comment:
Do we need both figures?
We only need to sort by % here.

Reply:
We need both figures here to calculate the percentage. The percentage function `percentage(count_by_period[period], overdue_patients(row, period))` returns the percentage

Reply:
It's a function that gets the percentage using 2 figures. So essentially just the percentage